Lib/test/test_buffer.py

```diff
@@ -4436,5 +4436,95 @@ def test_pybuffer_size_from_format(self):
                              struct.calcsize(format))
 
 
+class TestPythonBufferProtocol(unittest.TestCase):
+    def test_basic(self):
+        class MyBuffer:
+            def __buffer__(self, flags):
```

Comment:
There are no tests that actually use `flags` or check their type / contents. 

Is it intentional?

Reply:
Good point, will add some. I think the tests aren't very comprehensive, so any suggestions for new tests are appreciated.

Reply:
Added.